Plots/Skew-T/NCL_skewt_1.py

```diff
@@ -0,0 +1,124 @@
+"""
+NCL_skewt_1.py
+===============
+This script illustrates the following concepts:
+   - Drawing a default Skew-T background
+   - Customizing the background of a Skew-T plot
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/skewt_1.ncl
+    - Original NCL plots: https://www.ncl.ucar.edu/Applications/Images/skewt_1_1_lg.png and https://www.ncl.ucar.edu/Applications/Images/skewt_1_2_lg.png and https://www.ncl.ucar.edu/Applications/Images/skewt_1_3_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import matplotlib.pyplot as plt
+import matplotlib.lines as mlines
+import numpy as np
+from metpy.plots import SkewT
+from metpy.units import units
+
+import geocat.viz.util as gvutil
+
+###############################################################################
+# Plot Skew-T with MetPy Defaults:
```

Comment:
```suggestion
# Plot Skew-T with MetPy Defaults:

```
Adding spacing between the top line and secondary comment lines for better sphinx generation